### PR TITLE
Adds almalinux build image support into tools

### PR DESCRIPTION
**Pull request by gurkanindibay**
This PR is related to https://github.com/citusdata/packaging/pull/1006
To make almalinux work, I needed to change tools.

### CODE REVIEW

File: packaging_automation/citus_package.py [hunk 1]
```diff
@@ -31,14 +31,19 @@
 POSTGRES_EXCLUDE_FILE_NAME = "pg_exclude.yml"
 
 docker_image_names = {
+    "almalinux": "almalinux",
+    "rockylinux": "almalinux",
     "debian": "debian",
+    "el/9": "almalinux-9",
```
Comment (gurkanindibay) [inline packaging_automation/citus_package.py:37]: Before this change, I was only using OS name to define docker image name. Sincei both centos and almalinux is being used as base image now, I added os version as well

File: packaging_automation/tests/test_citus_package.py [hunk 2]
```diff
@@ -28,6 +28,10 @@
     "el/8": 3,
     "ol/7": 2,
     "ol/8": 3,
+    "almalinux/9": 3,
+    "rockylinux/9": 3,
+    "elnew/9": 3,
+    "olnew/9": 3,
```
Comment (onurctirtir) [inline packaging_automation/tests/test_citus_package.py]: why do we have those "new" suffixes ?

File: packaging_automation/citus_package.py [hunk 3]
```diff
@@ -279,6 +285,8 @@ def get_release_package_folder_name(os_name: str, os_version: str) -> str:
 
 
 def get_docker_image_name(platform: str):
+    if platform in docker_image_names.keys():
+        return docker_image_names[platform]
```
Comment (onurctirtir) [inline packaging_automation/citus_package.py:293]: Maybe comment:
```py
# Use hardcoded docker image names (e.g., almalinux-9) not the generic ones
# (e.g., almalinux) if docker_image_names tells us to do so.
```

Comment (gurkanindibay) [reply packaging_automation/citus_package.py:293]: Detailed the comments for the method

File: packaging_automation/tests/test_citus_package.py (hunk 2 above)
Comment (gurkanindibay) [reply packaging_automation/tests/test_citus_package.py]: it was my fault. Removed the new suffixes.

**Review by onurctirtir — APPROVED** [review]

**Pull request merged**